Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -98,6 +99,7 @@ class BrowserViewController: UIViewController {
         self.profile = profile
         self.tabManager = tabManager
         self.readerModeCache = DiskReaderModeCache.sharedInstance
+        self.handoffManager = HandoffManager.sharedInstance
```

Comment:
If there is a shared instance of HandoffManager, we don't need to have a reference to it permanently in BrowserViewController. I would just replace all occurrences of `handoffManger` to `HandoffManager.sharedInstance`.